common/src/main/java/com/google/auto/common/MoreElements.java

```diff
@@ -189,5 +197,41 @@ public static boolean isAnnotationPresent(Element element,
     return Optional.absent();
   }
 
+  /**
+   * Returns a {@link Predicate} that can be used to filter elements by {@link Modifier}.
+   * The predicate returns {@code true} if the input {@link Element} has all of the given
+   * {@code modifiers}, perhaps in addition to others.
+   *
+   * <p>Here is an example how one could get a List of static methods of a class:
+   * <code>
+   * FluentIterable.from(ElementFilter.methodsIn(clazzElement.getEnclosedElements()))
+   *     .filter(MoreElements.hasModifiers(Modifier.STATIC).toList();
+   * </code>
+   */
+  public static Predicate<Element> hasModifiers(Modifier... modifiers) {
+    return hasModifiers(ImmutableSet.copyOf(modifiers));
+  }
+
+  /**
+   * Returns a {@link Predicate} that can be used to filter elements by {@link Modifier}.
+   * The predicate returns {@code true} if the input {@link Element} has all of the given
+   * {@code modifiers}, perhaps in addition to others.
+   *
+   * <p>Here is an example how one could get a List of methods with certain modifiers of a class:
+   * <code>
+   * Set&lt;Modifier> modifiers = ...;
```

Comment:
Sorry not to have caught this before. In Java 8, javadoc by default is a lot more finicky about the contents of HTML, and in particular I have noticed that it doesn't like unquoted `>` (even though most actual HTML renderers are fine with it). Also, I don't think the line breaks would be preserved in the text as written. I would suggest quoting the whole code block, something like this:
```
<pre>{@code
Set<Modifier> modifiers = ...;
FluentIterable.from(...etc...
}</pre>
```